### PR TITLE
DO-177: Added Image components for logo on App Header.

### PR DESCRIPTION
**Pull request by mohitsaladiya**

### CODE REVIEW

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
 .app-header__logo {
   width: 200px;
-  height: 70px;
   float: left;
   padding: 10px 10px 10px 20px;
   cursor: pointer;
   .image {
-    width: 100%;
+    height: 70px;
+    width: 180px;
```
Comment (amitjain24524) [inline src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss:10]: Image Width always set AUTO.

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
+    position: absolute;
+    background: var(--logo-url);
```
Comment (amitjain24524) [inline src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss:12]: Why we need background and background-size if we are using image.

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss:10]: Done

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/_AppHeaderDesktopMenu.scss:12]: It was not deleted by mistake.

**Pull request merged**